app/src/test/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorStateTest.kt

```diff
@@ -22,7 +23,7 @@ class CreditCardEditorStateTest {
         cardNumberLast4 = "1110",
         expiryMonth = 5,
         expiryYear = 2030,
-        cardType = "amex",
+        cardType = CreditCardNetworkType.AMEX.cardName,
```

Comment:
Leaving a comment here that cardType should really be the enum value. This is something that was discussed on the AC changes as well. The effects of not using that here means that consumers need to constantly convert from a non-type checked value whenever they need to use it.